### ISSUE TITLE
Group video calling

### ISSUE THREAD

**Issue by monikarora** [QuickBlox/quickblox-android-sdk#429]
Hey , i want to add group video calling in my app . I've checked code than compile successfully , but if i run code with my credential(create account in quickblox) its not working if i used in-built credential its working but always show inbuilt user listing . i don't know  how to use group video in my app , please help me 
Thanks!

**Comment by RomanPronin** [comment]
Hi @monikarora, what kind of inbuilt user listing are you talking about? In sample-videochat-webrtc you create user and room manually, like [here](https://github.com/QuickBlox/quickblox-android-sdk/blob/master/sample-videochat-webrtc/screenshots/Login.png) Look at sample more carefully, and if you get some error show us.

**Comment by monikarora** [comment]
Hey @RomanPronin , already create user than add room manually than show user list that listing not add by me . how to add my own contact list . please check these images
![list](https://user-images.githubusercontent.com/17878500/28667026-54f90a82-72e7-11e7-9dcb-c5777dc71696.png)
![login](https://user-images.githubusercontent.com/17878500/28667025-54f2e418-72e7-11e7-90ce-582a08d4f373.png)

**Comment by RomanPronin** [comment]
Yes, it is cause you are joined to existing room. Room name is a user tag, which you created ([link](https://quickblox.com/developers/SimpleSample-users-android#Sign_Up)). So, either create unique room-tag, or use credentials from your own account. Pay attention, for the proper work of the sample you have to [setup GCM](https://quickblox.com/developers/SimpleSample-messages_users-android#Setup_GCM) too.

**Comment by monikarora** [comment]
Okay , but own credentials not work . get credentials from this link https://admin.quickblox.com/account/settings  & https://admin.quickblox.com/apps/60721/edit , if i add my own credentials  app is not login

**Comment by RomanPronin** [comment]
Uninstall app completely, and then deploy it again. The problem might have been in app cache with old settings. If you still get error, then show it us.

**Comment by monikarora** [comment]
Okay i'll try again

**Comment by monikarora** [comment]
Hey @RomanPronin  thanks now its working . please tell me how to add user in to list . always hit signup api ?

**Comment by RomanPronin** [comment]
Yes, you should do signup to create user. So, use the same room name, on other device. One device, one user.

**Comment by monikarora** [comment]
Okay @RomanPronin  ,if app is not open video calling is not working ?

**Comment by RomanPronin** [comment]
@monikarora, if you want to get call, while app isn't running, you should set [push notifications](https://quickblox.com/developers/SimpleSample-messages_users-android#Setup_GCM). I advise you to go deeper into the sample code for understanding the principles of its work.

**Comment by monikarora** [comment]
Okay @RomanPronin  Thanks for help

**Comment by jay2342** [comment]
hey monikarora 
Hey RomanPronin Roman i need your help

**Comment by monikarora** [comment]
Hey @jay2342  yes tell me

**Comment by jay2342** [comment]
monikarora what can you make from android studio and can we be friends

**Comment by jay2342** [comment]
monikarora

**Comment by jay2342** [comment]
monikarora

**Comment by monikarora** [comment]
Hey @jay2342 tell me issue you have face for quickblox video chat ?

**Comment by jay2342** [comment]
monikaroa i have issue about quickblox android studio chat app

**Comment by nirjastaff** [comment]
i have issue whille chnage package name and  if of Quickbox of db_config.json file video calling screenn not show

**Comment by rizwanjillani** [comment]
Hi , @monikarora i m facing issue while saving session.
please help regarding how to save session

**Comment by QB-nikolay-zolotarov** [comment]
Hello QuickBlox customers,

This is Nikolay from QuickBlox support.

The issue was closed as it is outdated.

Please check the relevant section of our documentation here:
[https://docs.quickblox.com/docs/android-authentication](https://docs.quickblox.com/docs/android-authentication)

Also, please update the SDK to the latest version:
[https://github.com/QuickBlox/quickblox-android-sdk-releases/releases/tag/3.9.11](https://github.com/QuickBlox/quickblox-android-sdk-releases/releases/tag/3.9.11)

Additionally, please check our new samples:
[https://docs.quickblox.com/docs/code-samples#chat-samples](https://docs.quickblox.com/docs/code-samples#chat-samples)
[https://docs.quickblox.com/docs/code-samples#video-calling-samples](https://docs.quickblox.com/docs/code-samples#video-calling-samples)

If it is still relevant after reviewing the updated information, feel free to open a new issue.

Have a nice day.
